staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -549,8 +549,7 @@ func TestPatchResourceWithConflict(t *testing.T) {
 		startingPod: &example.Pod{},
 		changedPod:  &example.Pod{},
 		updatePod:   &example.Pod{},
-
-		expectedError: `Operation cannot be fulfilled on pods.example.apiserver.k8s.io "foo": existing 2, new 1`,
+		expectedPod: &example.Pod{},
```

Comment:
this test computes and sends a patch of `{"spec":{"nodeName":"there"}}` (`delta(startingPod, changedPod)`), which applies cleanly on top of `updatePod`, and should succeed in a retry, not fail (see PR description for removal of conflicts-with-delta-from-first-seen-current-object behavior).

As a follow-up, it would be good to refactor these tests to make it clearer what patch is actually being sent and tested